frontend/apps/staff/.env.example

```
@@ -3,6 +3,14 @@ NEXT_PUBLIC_BASE_URL=/kasittely
 TILAVARAUS_API_URL=http://localhost:8000
 RESERVATION_UNIT_PREVIEW_URL_PREFIX="http://localhost:3000/reservation-unit"
 FEEDBACK_URL=
-SENTRY_DSN=
+SENTRY_ORG=city-of-helsinki
+SENTRY_PROJECT=tilavarauspalvelu-staff-ui
+NEXT_PUBLIC_SENTRY_PROJECT=${SENTRY_PROJECT}
 SENTRY_AUTH_TOKEN=
-SENTRY_ENVIRONMENT=local
+SENTRY_ENABLE_SOURCE_MAPS=false
+NEXT_PUBLIC_SENTRY_TRACE_PROPAGATION_TARGETS=https://localhost:8000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SENTRY_TRACE_PROPAGATION_TARGETS key should go before the SENTRY_AUTH_TOKEN key

(UnorderedKey)